test/powershell/Common/TestHostCS.psm1

```diff
@@ -191,7 +196,7 @@ namespace TestHost
 
         public override void WriteLine(string value)
         {
-            Streams.ConsoleOutput.Add(value);
+            Streams.ConsoleOutput.Add("::"+value+":NewLine");

```

Comment:
Why not use `` `n``, or I guess since this is actually C#, `\n`? Then you don't need `:NoNewline`.